### ISSUE TITLE
WSL vscode server does not run code in background.

### ISSUE THREAD

**Issue by DubbeleJ** [microsoft/vscode#193453]
Does this issue occur when all extensions are disabled?: Yes (with the exception of the WSL plugin, which is part of this bug).

- VS Code Version: 
1.82.2
abd2f3db4bdb28f9e95536dfa84d8479f1eb312d
x64
- OS Version: 
Distributor ID: Ubuntu
Description:    Ubuntu 22.04.1 LTS
Release:        22.04
Codename:       jammy
WSL2 version on a Windows 10 installation.

Steps to Reproduce:

1. in WSL ubuntu environment run the command `code .`
2. VSCode will start in Windows environment using the WSL plugin.
Expected behaviour:
After performing the `code .`  command, vscode (server) will run in background and return allowing continued use the the cmd prompt.
Observed behaviour:
the `code .` command is blocking and we do not return to the command prompt. vscode has to be terminated to return to make the command prompt available again.

I have noticed this behavioural change after the automatic update that has been applied in the past week.

If any logs or other kind of information is required, please let me know what is required and how I could obtain the information.

Thanks!

**Comment by eraffel-MDSol** [comment]
This changed for me after an update yesterday

**Comment by aeschli** [comment]
Looks like my fix for https://github.com/microsoft/vscode/issues/192506 caused that

**Comment by ryancausey** [comment]
I've noticed the same problem after updating VSCode running on WSL:

```bash
$ cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=22.04
DISTRIB_CODENAME=jammy
DISTRIB_DESCRIPTION="Ubuntu 22.04.3 LTS"
```

**Comment by aeschli** [comment]
I'm not able to reproduce this issue with the latest pre-release version of WSL (2.0.0).
Can someone confirm? To get the version of WSL run `wsl --version`
(https://github.com/microsoft/WSL/releases)
````
WSL version: 2.0.0.0
Kernel version: 5.15.123.1-1
WSLg version: 1.0.57
MSRDC version: 1.2.4485
Direct3D version: 1.608.2-61064218
DXCore version: 10.0.25880.1000-230602-1350.main
Windows version: 10.0.22631.2338
````

**Comment by ryancausey** [comment]
I'm using the latest released version of WSL and it reproduces there:

```
wsl --version
WSL version: 1.2.5.0
Kernel version: 5.15.90.1
WSLg version: 1.0.51
MSRDC version: 1.2.3770
Direct3D version: 1.608.2-61064218
DXCore version: 10.0.25131.1002-220531-1700.rs-onecore-base2-hyp
Windows version: 10.0.22621.2283
```

**Comment by gmolkvk** [comment]
Can reproduce, same version of WSL as ryancausey

code --version gives

```
1.82.2
abd2f3db4bdb28f9e95536dfa84d8479f1eb312d
x64
```

**Comment by nonprofittechy** [comment]
You can also use `CTRL+z`, `bg`.

Experiencing same issue here on WSL2 and Ubuntu 22.04

**Comment by robertmonka** [comment]
I have the same issue.

**Comment by aeschli** [comment]
The fix is in the upcoming 1.83 release.

**Comment by gmolkvk** [comment]
It's fixed for me :party:
